qr_code/qrcode/utils.py

```diff
@@ -1,96 +1,198 @@
+                 version_dark=False, version_light=False, format_dark=False,
+                 format_light=False, alignment_dark=False, alignment_light=False,
+                 timing_dark=False, timing_light=False, separator=False,
+                 dark_module=False, quiet_zone=False):
+        """
+        :param size: The size of the QR code as an integer or a string.
+                    Default is *'m'*.
+        :type: str or int
+        :param int border: The size of the border (blank space around the code).
+        :param int version: The version of the QR code gives the size of the matrix.
+                Default is *None* which mean automatic in order to avoid data overflow.
+        :param str image_format: The graphics format used to render the QR code.
+                It can be either *'svg'* or *'png'*. Default is *'svg'*.
+        :param str error_correction: How much error correction that might be required
+                to read the code. It can be either *'L'*, *'M'*, *'Q'*, or *'H'*. Default is *'M'*.
+        :param bool micro: Indicates if a Micro QR Code should be created. Default: False
+        :param dark: Color of the dark modules (default: black). The
+                color can be provided as ``(R, G, B)`` tuple, as hexadecimal
+                format (``#RGB``, ``#RRGGBB`` ``RRGGBBAA``), or web color
+                name (i.e. ``red``).
+        :param light: Color of the light modules (default: white).
+                See `color` for valid values. If light is set to ``None`` the
+                light modules will be transparent.
+        :param finder_dark: Color of the dark finder modules (default: same as ``dark``)
+        :param finder_light: Color of the light finder modules (default: same as ``light``)
+        :param data_dark: Color of the dark data modules (default: same as ``dark``)
+        :param data_light: Color of the light data modules (default: same as ``light``)
+        :param version_dark: Color of the dark version modules (default: same as ``dark``)
+        :param version_light: Color of the light version modules (default: same as ``light``)
+        :param format_dark: Color of the dark format modules (default: same as ``dark``)
+        :param format_light: Color of the light format modules (default: same as ``light``)
+        :param alignment_dark: Color of the dark alignment modules (default: same as ``dark``)
+        :param alignment_light: Color of the light alignment modules (default: same as ``light``)
+        :param timing_dark: Color of the dark timing pattern modules (default: same as ``dark``)
+        :param timing_light: Color of the light timing pattern modules (default: same as ``light``)
+        :param separator: Color of the separator (default: same as ``light``)
+        :param dark_module: Color of the dark module (default: same as ``dark``)
+        :param quiet_zone: Color of the quiet zone modules (default: same as ``light``)
```

Comment:
I know this is consistent with Segno API, but I find it a little bit unclear without the word color (for instance as a suffix like data_light_color).